src/app/_constants/common.ts

```diff
@@ -23,7 +23,7 @@ export type CaseLink = {
 };
 
 // modals
-export const ADD_MODAL_MAX_WIDTH = 1110;
+export const ADD_MODAL_MAX_WIDTH = '1200';
```

Comment:
needs to be string?

Reply:
Yes, otherwise it is not working with "width", only with "maxWidth". Like this both versions are working fine.